### ISSUE TITLE
Create feature to allow approval requests to be reassigned. 

### ISSUE THREAD

**Issue by Steven-Chorkawy** [Steven-Chorkawy/Submit-AR-Invoice#65]
4)   When I reassigned to a different "accountant" and it still shows in   the Finance tab under "for you" and can be approved by me still. if   approver is removed then it shouldn't hold up approval. Is this the same for   reassigning managerial approval? It should be able to be reassigned and not   wait on the approval from someone who is not req'd
--

**Comment by Steven-Chorkawy** [comment]
out of scope of this project.
